docs/client.md

```diff
@@ -7,7 +7,7 @@ Please check [Quick Start](Quick Start/Stake.md) for installation instructions
 
 ## Commands
 Run the commands in following way:
-    `node index.js <command>`
+    `./razor <command>`
 You can run following commands in CLI:
 
 `-help` See a list of available commands
```

Comment:
Can you separate all the commands with a heading, possibly h3, like:
### Help
`./razor --help` 
something like this